src/path_resolver.h

```diff
@@ -3,6 +3,19 @@
 
 #include "uvwasi.h"
 
+/* TODO(cjihrig): PATH_MAX_BYTES shouldn't be stack allocated. On Windows, paths
```

Comment:
This was just recently removed. I don't think it should be added back. The next time uvwasi is updated in wasm3, anything related to `PATH_MAX` should go away.

Reply:
I am technically fixing an issue divided in 2 repositories `wasm3` and `uvwasi` so things got mixed up, okay so I will fix that from `wasm3`